BP/history/provinces/bp_k_lusitania.txt

```diff
@@ -1,150 +1,178 @@
-﻿# k_lusitania
+﻿#Notes:
+# Visigothic settlement regions in Hispania:
+# Mérida, Barcelona, Hispalis, Toledo, Rioja
+# Douro, Ebro, Tagus river basin and Tierra de Campos
+#
+# https://en.wikipedia.org/wiki/Visigothic_Kingdom#Visigothic_settlements
+# https://commons.wikimedia.org/wiki/File:Hispania3c.JPG
+#
+# Lusitanian distribution
+# https://en.wikipedia.org/wiki/Lusitanians#/media/File:Ethnographic_Iberia_200_BCE.PNG
```

Comment:
Ok, but this map is for 200 BCE. If Lusitanian remains, which I'm not really persuaded of, it is in remote corners.

Reply:
Abian36's changes add Lusitanian to c_coimbra; that 200 BC map shows Lusitanian existed further inland. Instead of removing it alltogether, I moved Lusitanian inland. I'm not vested either way on keeping Lusitanian culture, but I will note c_viseu is an all mountain county, so it might count as "remote" for the purposes of surviving until the Migration period.

Either way, I'll removed lusitanian from c_castelo_branco.